internal/build/imgsrc/resolver.go

```diff
@@ -108,10 +188,14 @@ func (r *Resolver) ResolveReference(ctx context.Context, streams *iostreams.IOSt
 		}
 		bld.BuildAndPushFinish()
 		bld.FinishImageStrategy(s, true /* failed */, nil, note)
+		span.AddEvent(fmt.Sprintf("failed to resolve image with strategy %s", s.Name()))
+
 	}
 
 	r.finishBuild(ctx, bld, true /* failed */, "no strategies resulted in an image", nil)
-	return nil, fmt.Errorf("could not find image \"%s\"", opts.ImageRef)
+	err = fmt.Errorf("could not find image \"%s\"", opts.ImageRef)
```

Comment:
Not a big deal but I recently discovered that the `%q` verb prints a quoted string. Much easier to read/type than escaped quotes :D